### PR TITLE
Adding Windows support

### PR DESCRIPTION
**Pull request by domesticmouse**
/cc @stuartmorgan and @csells

### CODE REVIEW

File: window_to_front/windows/window_to_front_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
+  // Replace "getPlatformVersion" check with your plugin's method.
+  // See:
+  // https://github.com/flutter/engine/tree/master/shell/platform/common/cpp/client_wrapper/include/flutter
+  // and
+  // https://github.com/flutter/engine/tree/master/shell/platform/glfw/client_wrapper/include/flutter
+  // for the relevant Flutter APIs.
+  if (method_call.method_name().compare("activate") == 0) {
+    result->Success(flutter::EncodableValue(BringWindowToTop(
+      registrar_->GetView()->GetNativeWindow())));
```
Comment (domesticmouse) [inline window_to_front/windows/window_to_front_plugin.cpp]: @stuartmorgan and @csells is there anything obviously wrong here? It compiles, but then doesn't bring the window to front. Sadly.

Comment (csells) [reply window_to_front/windows/window_to_front_plugin.cpp]: are you calling [SetForegroundWindow](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setforegroundwindow)? I can't tell...

Comment (domesticmouse) [reply window_to_front/windows/window_to_front_plugin.cpp]: Turns out https://stackoverflow.com/a/34414846/2142626 is the best answer I've found so far. And it's a doozy.

**Comment by domesticmouse** [comment]
Fixes: https://github.com/flutter/flutter/issues/65864

**Pull request merged**